public/assets/partials/schedule.partial-96f15f97a3655d4a8714c522c8d7747f9e3767255f240552fdbf4454c00e94d7.html

```diff
@@ -27,7 +27,7 @@ <h1 class="center-align underline">Schedule</h1>
     </div>
 
     <div class="col-xs-12 col-sm-6 col-md-4 col-sm-offset-4 panel-box border-bottom">
-      <strong>16:00 -</strong> Speaches/Toastes
+      <strong>16:00 -</strong> Speaches/Toasts
```

Comment:
This needs to be updated in `/app/assets/javascripts/partials/schedule.partial.html`